apps/nextjs-website/.storybook/preview.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import type { Preview } from '@storybook/react';
+import { ThemeProvider } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+import {
+  INITIAL_VIEWPORTS,
+  MINIMAL_VIEWPORTS,
+} from '@storybook/addon-viewport';
+import { theme } from '../src/app/theme';
+import './styles.css';
+
+const preview: Preview = {
+  parameters: {
+    viewport: {
+      viewports: {
+        ...INITIAL_VIEWPORTS,
+        ...MINIMAL_VIEWPORTS,
+      },
+    },
+    layout: 'fullscreen',
+    backgrounds: {
+      values: [
+        { name: 'light', value: '#FFFFFF' },
+        { name: 'dark', value: '#0B3EE3' },
+      ],
+    },
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/i,
+      },
+    },
```

Comment:
Are these necessary?

I don't think we have any colors or dates to be inserted and the matchers appear to be [the default ones](https://storybook.js.org/docs/essentials/controls#custom-control-type-matchers) anyway.